test/jasmine/tests/mapbox_test.js

```diff
@@ -449,7 +449,7 @@ describe('mapbox plots', function() {
 
             var divStyle = mapInfo.div.style;
             ['left', 'top', 'width', 'height'].forEach(function(p, i) {
-                expect(parseFloat(divStyle[p])).toBeWithin(dims[i], 5);
+                expect(parseFloat(divStyle[p])).toBeWithin(dims[i], 8);
```

Comment:
Oh nice. Does this make the full `npm run citest-jamine` run without errors for you?